src/views/Pathways/Milestones/Add/Milestone.js

```diff
@@ -4,32 +4,32 @@ import { withRouter } from 'react-router';
 import { bindActionCreators } from "redux";
```

Comment:
This file doesn't need to exist under `Pathways/Milestones/Add/Milestones.js`

it can simply be `Pathways/Milestones/Add.js`. The `Add.js` within Milestones.js signifies that it is related to milestones. Make sure such structure is followed everywhere. Not just in this file.